src/instructlab/sdg/generate_data.py

```diff
@@ -393,7 +637,6 @@ def generate_data(
 
         if samples[0].get("document"):
```

Comment:
This can use `is_knowledge`

Reply:
Yes, good catch! I missed that when rebasing on top of the latest mmlubench changes.